test/SimpleTest.php

```diff
@@ -2,15 +2,128 @@
 
 namespace Bschmitt\Amqp\Test;
 
+use Bschmitt\Amqp\Publisher;
+use \Mockery;
+use Illuminate\Config\Repository;
+
 /**
  * @author Björn Schmitt <code@bjoern.io>
  */
 class SimpleTest extends \PHPUnit_Framework_TestCase
 {
+    private $REPOSITORY_KEY = 'amqp';
```

Comment:
What do you think about a constant instead? This shouldn't change for the tests.